core/templates/dev/head/css/oppia.css

```diff
@@ -704,6 +704,22 @@ textarea {
   }
 }
 
+.navbar-nav.oppia-navbar-nav > li > a.oppia-navbar-editor-navigation-toggle{
+font-size: 30px;
+padding: 10px 15px;
+
+}
+
+.oppia-editor-navigation-dropdown > .active > a,.oppia-editor-navigation-dropdown > .active > a:focus,.oppia-editor-navigation-dropdown > .active > a:hover{
+  background-color: #009688 !important;
```

Comment:
In general, we try not to use !important, since this tends to make it unclear what has priority over what (especially if a whole bunch of rules do this). Can it be removed?